packages/remix-server-runtime/server.ts

```diff
@@ -1,3 +1,7 @@
+/// <reference lib="dom.iterable" />
```

Comment:
I do not like this and will explore more tomorrow. I think this is a byproduct of how we have configured our root tsconfig.